apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -900,17 +949,17 @@ func (m *ModelManagerAPI) getModelInfo(tag names.ModelTag, withSecrets bool) (pa
 			info.AgentVersion = &agentVersion
 		}
 
-		// TODO(stickupkid): Series is deprecated, always use a base as the
-		// source of truth.
-		defaultBase := config.PreferredBase(cfg)
-		info.DefaultBase = defaultBase.String()
-		if defaultSeries, err := corebase.GetSeriesFromBase(defaultBase); err == nil {
-			info.DefaultSeries = defaultSeries
-		} else {
-			logger.Errorf("cannot get default series from base %q: %v", defaultBase, err)
-			// This is slightly defensive, but we should always show a series
-			// in the model info.
-			info.DefaultSeries = jujuversion.DefaultSupportedLTS()
+		if withDefaultOS {
```

Comment:
Can you add a note, as to why we no longer want to return `DefaultBase`...

Reply:
DefaultBase is a model-config item, so this item is duplicated. Is a user wants to know the default base, this is where they should look

It's what is required to see the default-base in the CLI, `show-model` and `list-model` both do not show this

Reply:
Comment added in code